cmd/olfullnode/devnet.go

```diff
@@ -468,16 +482,30 @@ func initialState(args *testnetConfig, nodeList []node, option ethchain.ChainDri
 		}
 
 		pubkey, _ := keys.PubKeyFromTendermint(node.validator.PubKey.Bytes())
-		st := consensus.Stake{
-			ValidatorAddress: node.validator.Address.Bytes(),
-			StakeAddress:     stakeAddr,
-			Pubkey:           pubkey,
-			ECDSAPubKey:      h.PubKey(),
-			Name:             node.validator.Name,
-			Amount:           *vt.NewCoinFromInt(node.validator.Power).Amount,
+
+		if node.isValidator {
+			st := consensus.Stake{
+				ValidatorAddress: node.validator.Address.Bytes(),
+				StakeAddress:     stakeAddr,
+				Pubkey:           pubkey,
+				ECDSAPubKey:      h.PubKey(),
+				Name:             node.validator.Name,
+				Amount:           *vt.NewCoinFromInt(node.validator.Power).Amount,
+			}
+			staking = append(staking, st)
+		}
+
+		if node.isWitness {
```

Comment:
Are we sure we don't want to give a new token (something like WT) for witnesses. I understand that we don't need that right now. But probably it's a good idea to add a new dummy token WT or ( EWT, Ethereum Witness Token) for Eth witnesses, so that we can think of making It public down the line, somewhere in future